app/javascript/src/components/Dashboard/Notes/utils.js

```diff
@@ -1,13 +1,6 @@
-import dayjs from "dayjs";
-import relativeTime from "dayjs/plugin/relativeTime";
+import dayjs from "src/lib/dayjs";
```

Comment:
You don't have to import from this path all the time. That defeats the purpose of what we were doing. You can directly import from `dayjs` here.